src/components/calculator.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you edit the `calculator.js` component name to `Calculator.js` for [react components rules](https://stackoverflow.com/questions/30373343/reactjs-component-names-must-begin-with-capital-letters). It required that all React components must be **Capitalized** i.e **must start with an upper case letter**

Reply:
renamed the file to Calculator as requested